lib/util/index.coffee

```diff
@@ -128,8 +128,6 @@ exports.mergePineOptions = (defaults, extras) ->
 				if value?
 					if not isArray(value)
 						value = [value]
-					if !includes(value, 'id')
```

Comment:
Oh so now we will have to explicitly include the ID :+1: More deterministic result object structure is on the way :)

Reply:
Yep, it was just there before because lots of the object normalisation stuff needed it - now that's gone, this can go too :-)